src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+import { HomeAssistant } from "../../../types.js";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Condition {
+  entity: string;
+  state?: string;
+  state_not?: string;
+}
+
+interface Config extends LovelaceConfig {
+  card: LovelaceConfig;
+  conditions: Condition[];
+}
+
+class HuiConditionalCard extends LitElement implements LovelaceCard {
+  protected hass?: HomeAssistant;
+  protected config: Config;
+  protected card: LovelaceCard;
+
+  static get properties() {
+    return {
+      hidden: {},
+    };
+  }
+
+  protected render() {
+    return html`
+      <style>
+        .hidden {
+        display: none;
+        }
+      </style>
+      <div id="card"
+      class=${this.hidden ? "hidden" : ""}>
```

Comment:
classMap should be used here [Example](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/lovelace/cards/hui-glance-card.ts#L89)